python_src/pyproject.toml

```diff
@@ -77,5 +77,6 @@ max-line-length = 80
 min-similarity-lines = 10
 # ignore session maker as it gives pylint fits
 # https://github.com/PyCQA/pylint/issues/7090
-ignored-classes = ['sqlalchemy.orm.session.sessionmaker' ]
-ignore = ["migrations"]
+ignored-classes = ['sqlalchemy.orm.session.sessionmaker']
+# ignore the migrations directory. its going to have duplication and _that is ok_.
+ignore-paths = ["^src/lamp_py/migrations/.*$"]
```

Comment:
i don't think the `ignore` was doing anything before since the migrations dir didn't have an `__init__.py` file in it. once i added that (to get mypy to work properly), it started vomiting up issues. `ignore-paths` seems to work though.